src/main/java/store/MemoryProductStorage.java

```diff
@@ -0,0 +1,64 @@
+package store;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class MemoryProductStorage implements ProductStorage {
+
+	List<Produit> productList = new ArrayList<>();
```

Comment:
Pourquoi passer par une liste . La complexité de ce code est en o(n) au lieu de o(1) en utilisant une map:
http://javabypatel.blogspot.com/2015/10/time-complexity-of-hashmap-get-and-put-operation.html